pkg/pulumi/pulumi.go

```diff
@@ -18,21 +19,27 @@ import (
 	"github.com/pulumi/pulumi/sdk/v3/go/common/tokens"
 	"github.com/pulumi/pulumi/sdk/v3/go/common/workspace"
 	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+	user "github.com/tweekmonster/luser"
```

Comment:
This module is already included as an indirect from pulumi itself. Should I be updating out `go.mod` to include it as a direct depe?

Reply:
you can simply run `go mod tidy`, and it should take care of things automatically